swap/swap.go

```diff
@@ -188,6 +188,8 @@ const (
 	receivedChequePrefix   = "received_cheque_"
 	connectedChequebookKey = "connected_chequebook"
 	connectedBlockchainKey = "connected_blockchain"
+	lastSentChequeKey      = "last_sent_cheque"
```

Comment:
While I see why you are adding the lastSentChequeKey and lastReceivedChequeKey, their function is somewhat different as the other `keys` before, which created an initial confusion when I was reading this code. The other keys are for writing to/ reading from the database, while these ones are only used in one function.
You could define them inside the function and then have them as a const (perhaps) in the test suite. You could return two mappings (one for sent and one for received).

Reply:
these keys are used:
- in the `Cheques` function
- in the `PeerCheques` function
- in the swap test suite (almost 30 times)

so they would have to be defined in those 3 places. if they are going to have the same value in all of those places, what do we gain by defining them separately each time?

in fact, we introduce a new failure point: if they are changed in 1 place and not the others, it will cause bugs. and even then, you would have to tediously look for these constants elsewhere and change them there as well.

as a separate issue: i don't understand the comment about returning 2 mappings. can you please elaborate on that?

Reply:
i can't think of a good name for these `const`s (suggestions are welcome). 

maybe add the word `map` somewhere?

alternative is: place them somewhere else

Reply:
maybe something ending in `ObjectKey` or `ResponseKey`. Then it would also be clear.

Reply:
renamed using `ResponseKey`

Reply:
> as a separate issue: i don't understand the comment about returning 2 mappings. can you please elaborate on that?

I meant that you can make the function such:
```golang
func (s *Swap) Cheques() (sent map[enode.ID]*Cheque, received map[enode.ID]*Cheque, err error)
```
(named return variables just for sake of clarity).
Doing it this way will completely remove the need for having a `ResponseKey` and is in my opinion cleaner. 

Think about how you would use this function. In your implementation, you would need to do
```golang
cheques, err := swap.Cheques()
sentCheques =: cheques[sentChequeResponseKey]
``` 
to get the sent cheques. My proposal would allow this in a one-liner. Also, the developer who will use this function will know directly that the function returns sentCheques and receivedCheques, something which is not trivial from the function signature in the current state.

Reply:
thanks for the clarification.

regarding the keys: i really don't think having 2 more `const` values in the code will significantly hurt clarity here. 

if it helps, i am also moving these to another file under #1883.

regarding the usage of the `Cheques` function: it's meant to be a functionality called from outside (as an RPC). i don't expect it to be internally used _anywhere_ in the code.

also:  consider that cheques are first grouped by peer, then marked as `sent` or `received`. so neither this (current) implementation nor your suggestion would allow someone to get all sent cheques in 1 (or 2) lines.

Reply:
also: an remote procedure call can only return **1** value (and an error). 

so it would be impossible to return the `sent` and `received` cheques separately as you suggest.

Reply:
@mortelli , thanks!

Reply:
Thank you for the clarification as well!